x/ibc/04-channel/keeper/timeout_test.go

```diff
@@ -0,0 +1,238 @@
+package keeper_test
+
+import (
+	"fmt"
+
+	connectionexported "github.com/cosmos/cosmos-sdk/x/ibc/03-connection/exported"
+	"github.com/cosmos/cosmos-sdk/x/ibc/04-channel/exported"
+	"github.com/cosmos/cosmos-sdk/x/ibc/04-channel/types"
+	commitment "github.com/cosmos/cosmos-sdk/x/ibc/23-commitment"
+	ibctypes "github.com/cosmos/cosmos-sdk/x/ibc/types"
+)
+
+func (suite *KeeperTestSuite) TestTimeoutPacket() {
+	counterparty := types.NewCounterparty(testPort2, testChannel2)
+	var (
+		packet      types.Packet
+		nextSeqRecv uint64
+		proofHeight uint64 = 3
+	)
+
+	testCases := []testCase{
+		// {"success", func() {
```

Comment:
Do we still not have a working success case here?

Reply:
not yet